plugins/modules/aci_interface_policy_leaf_policy_group.py

```diff
@@ -386,6 +386,10 @@ def main():
     state = module.params.get('state')
     name_alias = module.params.get('name_alias')
 
+    aci = ACIModule(module)
+    if lag_type == 'leaf' and port_channel_policy != 'null':
+      aci.fail_json('port_channel_policy is not a valid parameter for leaf(leaf-access port policy group), if used assign "null" to it(port_channel_policy = null).')
```

Comment:
add spaces before the ( in the text and also remove the - and add a space between leaf-access.